styleguide/source/assets/scss/01-atoms/_display-switch.scss

```diff
@@ -0,0 +1,22 @@
+.ama__display-switch {
+  display: inline-block;
+  border: solid 1px $purple;
+  border-radius: 5px;
+  padding: 1px 5px;
+  white-space: nowrap;
+  margin-right: $gutter / 4;
+  margin-bottom: $gutter / 4;
+  font-weight: 600;
+
+
+  &:hover {
+    cursor: pointer;
+  }
+
+  &--active {
+    background: $purple;
```

Comment:
probably should use background-color:

Reply:
I don't foresee us having to specify any other background properties for this element so I'm comfortable using the shorthand here. It can always be changed later if needed.